lib/components/ConflictPicker.vue

```diff
@@ -425,6 +450,14 @@ export default Vue.extend({
 		opacity: .5;
 		filter: saturate(.7);
 	}
+
+	:deep(.dialog__actions) {
```

Comment:
Sounds like a fix for NcDialog needed, there should be no need to use `deep`. Probably need to make actions flex without margin.
Then you could use `flex: 0 1 fit-content` on the buttons here and `flex: 1 1` on the separator and no `deep` styles needed.

Reply:
Well, I think the NcDialogs never had a left Button in mind when designed.
Not sure this is something we cant to push forward.

So yeah, until then, I'll keep it here

Reply:
yes was more of "loud thinking" for future tasks of mine^^